libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/ScreenLock/ScreenLockManager.swift

```diff
@@ -123,7 +155,7 @@ public class ScreenLockManager: NSObject {
         
         userAccounts?.forEach({ account in
             let hasMobilePolicy = account.idData.mobileAppPinLength > 0 && account.idData.mobileAppScreenLockTimeout != -1
-            self.storeMobilePolicy(userAccount: account, hasMobilePolicy: hasMobilePolicy)
+            self.storeMobilePolicy(userAccount: account, hasMobilePolicy: hasMobilePolicy, lockTimeout: account.idData.mobileAppScreenLockTimeout)
```

Comment:
Should we extend this out to Mobile SDK 12.0?   If a user is on `10.1.0` and goes straight to `11.0` they will miss this upgrade.

Reply:
Is this something that would be revisited at the time of MobileSDK 12? “If a user is on X and goes straight to Y…” may hold over time, until a baseline version is established. We know that SDK upgrades lag significantly.

Reply:
@bcriscuolo-sf  No.  This is the exact reason why apps are not allowed to skip major versions.  You cannot upgrade from 9.X to 11.X without upgrading users to some 10.X version in between.  You have an entire year between major releases, there is absolutely no excuse.

Reply:
@brandonpage EXCELLENT. Fully justified there. Is that documented somewhere (not that it will be a problem for SFS ever, but rather so "you" don't have to explain it again)?